pyuvdata/uvcal/calfits.py

```diff
@@ -125,7 +125,7 @@ def write_calfits(
             sechdr["EXTNAME"] = "FLAGS"
         # Conforming to fits format
         prihdr["SIMPLE"] = True
-        prihdr["BITPIX"] = 32
+        prihdr["BITPIX"] = -64
```

Comment:
Is this better than just not setting the BITPIX keyword at all and letting astropy handle it?

Reply:
I think this is better -- before, there was no way to know if astropy would set it to -64 or -32. I think this should be more consistent.

Reply:
ok. I think astropy uses the data types to assign the correct thing. I guess the only thing I'm worried about is promoting a the data artificially and therefore leading to larger files than necessary.